### ISSUE TITLE
Is there a way to get actual bounding box example of evaluation results?

### ISSUE THREAD

**Issue by guysoft** [facebookresearch/detr#181]
## ❓ How to do something using DETR

I ran detr on a custom dataset and ran the evaluation. I also ran the code in #106 and got a table of the results.

But how do I actually get the bounding boxes for the detection/evaluation of the results? 
I want to be able to see that is actually works, perhaps even use it, not just a metric.

How to get actual bounding box example of evaluation results?

NOTE:
Related  #106

**Comment by fmassa** [comment]
@guysoft we don't store the predictions in the `engine.py`, but you could modify it yourself to save the predictions in https://github.com/facebookresearch/detr/blob/master/engine.py#L123 for example.

Additionally, you can also use one of our [provided colab notebooks](https://colab.research.google.com/github/facebookresearch/detr/blob/colab/notebooks/detr_attention.ipynb) to visualize the model that you trained on a custom dataset.

I believe I've answered your question, and as such I'm closing this issue but let us know if you have further questions.

**Comment by guysoft** [comment]
Hey, so I used the notebook you provided above (detr_attention.ipynb), to run a prediction.
However - how do I load my own weights?

I have changed [based on following documentation](https://pytorch.org/tutorials/beginner/saving_loading_models.html#saving-loading-a-general-checkpoint-for-inference-and-or-resuming-training):
```python
model = torch.hub.load('facebookresearch/detr', 'detr_resnet50', pretrained=True)
model.eval();
```

To:
```python
model = torch.hub.load('facebookresearch/detr', 'detr_resnet50', pretrained=True)
path ="output/checkpoint.pth"
checkpoint = torch.load(path)
print(checkpoint["model"].keys())
model.load_state_dict(checkpoint['model_state_dict'])
model.eval();
```

But it seems there is no ``model_state_dict``. Where are the weights stored in the checkpoint?
``output/checkpoint.pth`` does load and has these keys:

```
dict_keys(['model', 'optimizer', 'lr_scheduler', 'epoch', 'args'])
```

Thanks for the pointers

**Comment by alcinos** [comment]
Replace
```
model.load_state_dict(checkpoint['model_state_dict'])
```
by
``` 
model.load_state_dict(checkpoint['model'])
```

**Comment by guysoft** [comment]
That works, thanks!
